### DISCUSSION TITLE
github pages not showing in my setting

### DISCUSSION THREAD

**Discussion by Akhi37918** [community/community#36942] [Other Feature Feedback, Questions, & Ideas]
no option found in setting

**Comment by DSDmark** [comment]
Hey @Akhi37918,

There are two ways to see or control your **github-page** 
Here is a **first** way how to see or control **git-page** by steps:-

 Step 1: Open your repo and if your **page** is already deployed. In that case, you can see **Environments**. 
 - ![Screenshot from 2022-10-22 14-08-27](https://user-images.githubusercontent.com/85744816/197330195-274abf2c-6ab5-4602-897a-a64725e5ebbd.png)
 
----
in that **Environments** section. There is a **github-page** section clink on it and after that it's look this :- 

- ![Screenshot from 2022-10-22 14-11-14](https://user-images.githubusercontent.com/85744816/197330389-f5d2e5c4-cc15-448b-9a90-648d62a02201.png)


And then click on **View deployment** button. You can see your page.

**Comment by DSDmark** [comment]
Hey @Akhi37918,
Here is a **second** way how to see or control **git-page** by steps:-

Step 1: Open your repo and on your **repo** open setting there ok. like this:-
 - ![Screenshot from 2022-10-22 14-24-21](https://user-images.githubusercontent.com/85744816/197330718-68833bff-fdc2-464b-a914-73884108a541.png)

---
Step 2: inside the setting you can see **page** option in **Code and automation** section, click on it.

- ![Screenshot from 2022-10-22 14-25-55](https://user-images.githubusercontent.com/85744816/197330804-7fdc800e-979d-4e5b-a250-fd7225a85e35.png)

Step 3: Now you are able to **delete**, **republish** and **unpublish** your **git-page**

**Reply by DSDmark** [reply]
I forget to mention I see your project, but I can't find a single **git page**, so you need to set up your page first, ok.

**Reply by Akhi37918** [reply]
git page not showing in settings

**Reply by Akhi37918** [reply]
![image](https://user-images.githubusercontent.com/116263412/197355324-b0948ece-439f-4b9e-99cf-6b1b589d96e4.png)
 no option for web hosting

**Reply by Akhi37918** [reply]
![image](https://user-images.githubusercontent.com/116263412/197355468-1e26d597-9da7-4171-8f2c-3a27301b4073.png)

**Reply by Akhi37918** [reply]
tnku bro now working

**Reply by DSDmark** [reply]
Hey @Akhi37918,
I hope you get your answer. If you get your answer, then you should close that question by click **accept answer** button, or you can write the answer and accept own answer.
